cardano-db-sync/src/Cardano/DbSync/Era/Universal/Epoch.hs

```diff
@@ -214,7 +214,7 @@ insertEpochStake syncEnv nw epochNo stakeChunk = do
       (StakeCred, (Shelley.Coin, PoolKeyHash)) ->
       ExceptT SyncNodeError (ReaderT SqlBackend m) DB.EpochStake
     mkStake cache (saddr, (coin, pool)) = do
-      saId <- lift $ queryOrInsertStakeAddress trce cache UpdateCache nw saddr
+      saId <- lift $ queryOrInsertStakeAddress trce cache UpdateCacheStrong nw saddr
```

Comment:
how can one tell which update strategy they should pick? I can see we have two, but can't work out why some go the LRU whilst others go to the Map.